js/script.js

```diff
@@ -0,0 +1,15 @@
+function getTriangleArea (a, h) {
+  if ( (a <= 0) || (h <= 0) ) {
+    console.log('Nieprawidłowe dane');
+  } else if ( (a >= 0) && (h >= 0) ) {
```

Comment:
when you add return in first if you don't need else - next return will be enough